application/modules/home/controllers/Home.php

```diff
@@ -43,4 +43,19 @@ public function index() {
 		// load view
 		$this->load_view('top', $this->data);
 	}
-}
\ No newline at end of file
+
+	/**
+	 * Test side ci
+	 *
+	 * @return void
+	 */
+	public function side_ci()
+	{
+		$this->data['exhibition']['name'] = 'test';
+
+		// load view
+		$this->load_view('side_ci', $this->data);
+	}
+}
+// End of file side_ci.php
```

Comment:
__[code_sniffer]__
* Expected 1 newline at end of file; 0 found
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment

<!-- Comment by [SideCI](https://sideci.com) -->